src/build.sh

```diff
@@ -365,6 +400,10 @@ function parse_build_options()
         options_values['FULL_CLEANUP']=1
         shift
         ;;
+      --cflags)
+        options_values['CFLAGS']="$2"
```

Comment:
In your code, it looks like you did not validate if this flag is passed empty.

Reply:
Hi @rodrigosiqueira. I don't understand, if no value is passed, the build itself gives an error due to this part of the code:
```
if [[ "$?" -gt 0 ]]; then
  complain "Invalid option: ${options_values['ERROR']}"
  build_help
  exit 22 # EINVAL
fi
```